src/params/grid_constants_cbl.F90

```diff
@@ -45,12 +46,16 @@ MODULE grid_constants_mod_cbl
 INTEGER, PARAMETER :: nsCs      = 2  ! # soil carbon stores
 INTEGER, PARAMETER :: nvCs      = 3  ! # vegetation carbon stores
 INTEGER, PARAMETER :: ICE_SoilType = 9 ! SoilType Index (soilparm_cable.nml JAC)
-INTEGER, PARAMETER :: lakes_cable  = 16! SoilType Index (soilparm_cable.nml JAC)
-
-INTEGER, PARAMETER :: ICE_VegType = 17  ! permanent ice index for veg
-
-INTEGER, PARAMETER :: mf = 2          ! # leaves (sunlit, shaded)
-INTEGER, PARAMETER :: niter = 4       ! number of iterations for za/L
+#ifndef UM_CBL
+INTEGER, PARAMETER :: lakes_cable  = 16 ! cable_surface_type (nml) Index 
+INTEGER, PARAMETER :: ice_cable    = 17 ! cable_surface_type (nml) Index
+#endif
```

Comment:
It will be easier to change the offline side. jules_surface_types have an ice, urban, etc so on the CABLE side (cable_surface_types) already in JAC it is, called ice_cable, urban_cable. Possibly not even necessary there, but easier to change on our side.